packages/synapse-react-client/src/components/Authentication/UsernamePasswordForm.tsx

```diff
@@ -13,9 +12,7 @@ type UsernamePasswordFormProps = {
 
 export default function UsernamePasswordForm(props: UsernamePasswordFormProps) {
   const {
-    resetPasswordUrl = `${getEndpoint(
-      BackendDestinationEnum.PORTAL_ENDPOINT,
-    )}PasswordReset:0`,
+    resetPasswordUrl = `https://accounts.sagebionetworks.synapse.org/resetPassword`,
```

Comment:
We will have to think about how we want to handle these links for each stack:
1. Should these links ensure you maintain in contact with the same backend environment?
2. Should links on e.g. staging.synapse.org send you to the staging accounts site, or always prod?